docs/core/reference/discounts.md

```
@@ -74,29 +74,30 @@ For performance reasons the applicable discounts are cached per request. If you
-|:-------------------|:------------------------|:------------------------------|
-| `id`               |                         |                               |
-| `discount_id`      |                         |                               |
-| `purchasable_type` |                         | `product_variant` 
-| `type`             | `condition` or `reward` |
-| `created_at`       |                         |                               |
-| `updated_at`       |                         |                               |
+| Field               | Description             | Example                       |
+|:--------------------|:------------------------|:------------------------------|
+| `id`                |                         |                               |
+| `discount_id`       |                         |                               |
+| `discountable_type` |                         | `product_variant` 
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)